### ISSUE TITLE
Unable to download videos and watch successfully

### ISSUE THREAD

**Issue by nusbe-54** [aniyomiorg/aniyomi-extensions#3360]
### Source information

Aniwave 14.69

### Source language

English

### Steps to reproduce

1. Try to download Go Go Loser ranger from episode 3 onwards (1 and 2 downloaded fine on the same day and were able to watch with no issues).
2. For example, when trying to download episode 3, it will download it (shows progress), but once it is done downloaded it will display an error icon in the app and the notification bar will have a message: 'Expected URL scheme http or https but no scheme for /tree/..., retrying..'

### Expected behavior

It should download successfully and watch as normal.

### Actual behavior

It won't download the video, even after turning on a VPN or even trying to change user agent, then it will show 'No video found'. Funny enough, I can stream the videos and they will play just fine, but cannot download them!

### Aniyomi version

0.15.3.0

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.2.4](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/docs/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Secozzi** [comment]
We don't support the aniwave extension anymore. See #3341
